findTheMiddleElement/solution.js

```diff
@@ -0,0 +1,3 @@
+var gimme = function (inputArray) {
```

Comment:
It makes me laugh that your solution is so functional/arrow-y, but you still declare the function using old syntax! You could write this all on one line as:

`var gimme = inputArray => inputArray.indexOf...`